runner/ansible/vars/azure_env.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+expected:
+  "1.1.1": "30000"
+  "1.2.1": "true"
```

Comment:
OK

Reply:
We need to have true as "true" here since the test code output true to stdout and we need to compare the string output. We could code around that, but it would be best to keep all then whens as:
  changed_when: config_updated.stdout != expected[test_id]

Reply:
Ok, it is ugly as hell, but I understand the reason